chapters/annotations.tex

```diff
@@ -1420,13 +1420,13 @@ \subsection{Version Numbering}\label{version-numbering}
 Version numbers are of the forms:
 \begin{itemize}
 \item
-  Main release versions: \lstinline!""" UNSIGNED-INTEGER { "." UNSIGNED-INTEGER } """!\\
+  Main release versions: \lstinline[language=grammar]!""" UNSIGNED-INTEGER { "." UNSIGNED-INTEGER } """!\\
```

Comment:
Well, well, this is what happens when reviewing one commit at a time. :)

Reply:
Yes, sorry for the confusion.